src/main/kotlin/no/nav/helse/rapids_rivers/KtorBuilder.kt

```diff
@@ -44,7 +51,7 @@ class KtorBuilder {
         builder.module(module)
     }
 
```

Comment:
Hadde det  vært en idé å eksponerer `build` for en kan spesifisere engine selv? 
ala

` fun build() = build(CIO)`

   ` fun <TEngine : ApplicationEngine, TConfiguration : ApplicationEngine.Configuration> build(factory: ApplicationEngineFactory<TEngine, TConfiguration>): ApplicationEngine = embeddedServer(factory, applicationEngineEnvironment {........`

Reply:
CIO støtter ikke http/2 feks . Men det kan jo løses ved å lage en egen ktor app ved siden av rapiden

Reply:
Commitet et forslag i https://github.com/navikt/rapids-and-rivers/pull/35/commits/8d1a1f823c87cfca22e0e66bce9dd9abf7509c44 
Kan reverte hvis det ikke gir mening

Reply:
Syntes det maker sense. Nå bør man vel i utgangspunktet ikke trenger så voldsom HTTP-server i en R&R-app, men vi har jo sett noen caser.

Og når det først er en slik escape-hatch så folk kan bytte tilbake til Netty er iallefall jeg mye mindre redd for å trykke på den grønne knappen.